core/types/src/svn.rs

```diff
@@ -59,6 +59,7 @@ impl ConfigId {
     }
 }
 
+// Pure array type larger than 32 so must implement default at the newtype level
```

Comment:
Since this is currently the only pure array wrapped, I chose not to do a macro yet...